packages/auth/amplify_auth_cognito_dart/lib/src/model/session/cognito_session_options.dart

```diff
@@ -13,25 +13,15 @@ class CognitoSessionOptions extends AuthSessionOptions
     with AWSEquatable<CognitoSessionOptions>, AWSDebuggable {
   /// {@macro amplify_auth_cognito.model.cognito_session_options}
   const CognitoSessionOptions({
-    this.getAWSCredentials = false,
```

Comment:
Thinking this over some more. Can we leave this in the constructor and mark it as deprecated with directions to just remove it, that it no longer does anything?

I'm worried a bit about the number of breaking changes and how this one, in particular, could be confusing.